packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -438,22 +439,23 @@ export = {
           mesh,
           listeners: [appmesh.VirtualNodeListener.grpc({
             port: 80,
-            tlsCertificate: appmesh.TlsCertificate.acm({
-              certificate: cert,
-              tlsMode: appmesh.TlsMode.STRICT,
-            }),
+            tls: {
+              mode: TlsMode.STRICT,
+              certificate: appmesh.TlsCertificate.acm({
+                certificate: cert,
+              }),
+            },
           },
           )],
         });
 
         // THEN
-
```

Comment:
This should not be changed.

Reply:
This original spacing is odd and other locations don't have spacing after `// Then` comment. Although it is not specifically related to this code change, I would like to make a change for it.

Reply:
OK.